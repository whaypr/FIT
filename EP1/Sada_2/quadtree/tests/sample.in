16
QQWBBWQWBBWQWBBWQWBBW


// THIS STUFF ARE JUST ANOTHER TESTS BUT USED IS ONLY THE FIRST ONE
8
QQQWWBBQWWBQBBWQWBWWBBQBBBWQBQWWBQBBWWBWQWWBW

8
QQQWBBWQWBBWQWBBWQWBBWQQWBBWQWBBWQWBBWQWBBWQQWBBWQWBBWQWBBWQWBBWQQWBBWQWBBWQWBBWQWBBW
