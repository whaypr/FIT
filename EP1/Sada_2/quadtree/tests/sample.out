#define quadtree_width 16
#define quadtree_height 16
static char quadtree_bits() = {
0xf0,0xf0,
0xf0,0xf0,
0xf0,0xf0,
0xf0,0xf0,
0x0f,0x0f,
0x0f,0x0f,
0x0f,0x0f,
0x0f,0x0f,
0xf0,0xf0,
0xf0,0xf0,
0xf0,0xf0,
0xf0,0xf0,
0x0f,0x0f,
0x0f,0x0f,
0x0f,0x0f,
0x0f,0x0f,
};
